gl/src/lib.rs

```diff
@@ -27,12 +27,20 @@ use std::time::Duration;
 
 pub struct GLDevice {
     version: GLVersion,
+    default_framebuffer_id: GLuint,
```

Comment:
uber-nit: "object" is the preferred GL terminology over "id"

Reply:
I renamed to `default_framebuffer`.